src/test/java/com/adyen/CheckoutTest.java

```diff
@@ -196,6 +204,25 @@ public void TestPaymentMethodsFailureMissingIdentifierOnLive() throws Exception
 
     }
 
+    @Test
```

Comment:
I think it will be a good idea to add one more test case for DefaultPaymentDetails class coverage.

Reply:
The intention here is to test the inject-ability of custom method details classes that implement PaymentMethodDetails. Generally we don't test model POJO classes and DefaultPaymentDetails is one of them, but feel free to send a suggestion of the extra test case if needed

Reply:
Testing PaymentMethodDetails is done at L211 and inject-ability of custom details class on L215.
Do you suggest to add one more test case or to remove the test case of L215 ? Could you give example?

Reply:
If it is already covered in L211 then we are good to go :)